app/jobs/ingest_mets_job.rb

```diff
@@ -78,4 +87,42 @@ def mets_to_repo_map
       @mets_to_repo_map ||= {}
     end
   end
+
+  class MVWIngester < Ingester
```

Comment:
Not having a MultiVolumeWork class makes "MVW" pretty obscure — can we call this something more transparent like HierarchicalIngester?